farcaster/hubs/values.yaml

```diff
@@ -44,13 +41,14 @@ controllers:
     dependsOn: hubble
     replicas: 1
     strategy: Recreate
-    image:
-      repository: graphiteapp/graphite-statsd
-      tag: 1.1.10-5
- 
+    containers:
```

Comment:
statsd controller also needs to follow the `controllers.{name}.containers.{name}` format

Reply:
had to remove dependsOn here as it's not allowed per common chart schema